src/modules/users/edit-user/edit-user.js

```diff
@@ -65,7 +65,10 @@
                                     vm.errorMsg = "Please enter a valid Email Id";
                                 }
                             } else if (keys.indexOf("name") !== -1) {
-                                vm.errorMsg = "Name is too short (minimum is 4 characters).";
+                                if(messages.indexOf("is too short (minimum is 4 characters)") !== -1)
+                                    vm.errorMsg = "Name is too short (minimum is 4 characters).";
+                                else if(messages.indexOf("is too long (maximum is 20 characters)") !== -1)
+                                    vm.errorMsg = "Name is too long (maximum 20 characters).";
```

Comment:
@umangachapagain  you have missed "is" in the error message. It should be - Name is too long (maximum is 20 characters).